app/src/main/res/layout/all_topics.xml

```diff
@@ -16,15 +16,14 @@
 
     <TextView
       android:id="@+id/all_topics_text_view"
-      android:layout_width="match_parent"
+      android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginStart="24dp"
-      android:layout_marginTop="16dp"
-      android:layout_marginEnd="24dp"
-      android:layout_marginBottom="16dp"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="65dp"
+      android:layout_marginBottom="24dp"
+      android:fontFamily="sans-serif-medium"
```

Comment:
Incorrect font, it should be roboto-medium

Reply:
This was suggested by Ben 
 See: https://stackoverflow.com/a/34445295/3689782.

Reply:
Sounds great, I wa not aware of this similarity.